UnityProject/Assets/Scripts/Tool/Multitool.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Multitool : MonoBehaviour, ICheckedInteractable<PositionalHandApply>
+{
+
+	public APC APCBuffer;
+
+	public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
+	{
+		// Use default interaction checks
+		if (!DefaultWillInteract.Default(interaction, side)) return false;
+		if (!Validations.IsTarget(gameObject, interaction))
```

Comment:
interactions don't always have a target object, such as when clicking space / floor. Add
`Validations.hasTarget(interaction)` to ensure it does. This causes NRE when clicking floor otherwise.